string			name
sensor_msgs/Image       image
---
int8			status
